.github/workflows/registry-scanner.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Scan Registry
-        uses: boostsecurityio/scanner-registry-action@v1.1.0
+        uses: boostsecurityio/scanner-registry-action@f68a83c176ae7feed63d86767a582b7ce41fe872
```

Comment:
note: this commit is in https://github.com/boostsecurityio/scanner-registry-action/pull/34